package.json

```diff
@@ -58,5 +59,22 @@
     "sequelize-test-helpers": "^1.1.2",
     "sinon": "^7.4.1",
     "sinon-chai": "^3.3.0"
+  },
+  "nyc": {
```

Comment:
Why do we have this here?

Reply:
I have no idea. @torsami77  must have put it there.

Reply:
Yes @sa-ma , the nyc config helps direct nyc on files and folders to include and exclude when calculating the code coverage percentage, as we have redundant lines of html templates that could reduce code coverage

Reply:
I think that's why we have `codeclimate.yml`